doc/classes/Transform2D.xml

```diff
@@ -183,6 +183,14 @@
 			<argument index="0" name="index" type="int">
 			</argument>
 			<description>
+		<method name="looking_at">
```

Comment:
The closing tags for the previous method is missing here. You're getting an error in static check:
`ERROR: Parse error reading file 'doc/classes/Transform2D.xml': mismatched tag: line 240, column 3`

Also, `looking_at` and `set_rotation` methods are not exposed to gdscript, if you add them to the documentation they need to be added here too:
https://github.com/godotengine/godot/blob/e0a65d8fe9a6e433c81fee8d6f256a4f03b37b2f/core/variant/variant_call.cpp#L1349-L1363